### PR TITLE
Search

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: events/consts.go [hunk 1]
```diff
@@ -35,4 +35,10 @@ const (
 
 	E_PUBLISHER_CREATE = "PUBLISHER_CREATE"
 	E_PUBLISHER_UPDATE = "PUBLISHER_UPDATE"
+
```
Comment (edoshor) [inline events/consts.go]: Needs `DELETE` events for blog_posts and tweets

File: consts/consts.go [hunk 2]
```diff
@@ -47,6 +47,7 @@ const (
 	CT_SONG                  = "SONG"
 	CT_BOOK                  = "BOOK"
 	CT_BLOG_POST             = "BLOG_POST"
```
[the remark this answers is not in the document]
Comment (YuriHechter) [reply consts/consts.go]: Done

File: es/blog_index.go [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+		qm.OrderBy("id"), // Required for same order results in each query
+		qm.Offset(offset),
+		qm.Limit(limit)).Bind(&posts)
+	if err != nil {
+		log.Errorf("indexPost error at offset %d. error: %v", offset, err)
+		return errors.Wrap(err, "Fetch blog posts from mdb.")
+	}
+	log.Infof("Adding %d blog posts (offset %d).", len(posts), offset)
+	for _, post := range posts {
+		/*if post.WPID == 160504 {
```
[the remark this answers is not in the document]
Comment (YuriHechter) [reply es/blog_index.go]: Done

File: events/consts.go (hunk 1 above)
Comment (YuriHechter) [reply events/consts.go]: Done

**Pull request merged**